plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePageSource.java

```diff
@@ -80,12 +80,12 @@ public HivePageSource(
             Optional<BucketValidator> bucketValidator,
             Optional<ReaderProjectionsAdapter> projectionsAdapter,
             TypeManager typeManager,
-            HiveTimestampPrecision timestampPrecision,
+            CoercionContext coercionContext,
```

Comment:
This belongs to previous commit where you introduced `coercionContext`

Reply:
I have squashed the commit